app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Here is the main page for a given user</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag "styles" %>
+  </head>
+
+  <body>
+    <div class='container'>
+      <div class='user-card'>
+        <%= render partial: 'shared/user_card', locals: { user: @user } %>
+      </div>
+
+      <div class='bio'>
+        <p><strong>Bio</strong></p>
+        <p><%= @user.bio %></p>
+      </div>
+
+      <div>
+        <% @user.posts.each do |post| %>
```

Comment:
- [ ] Looking at the wireframe, what is displayed are the 3 most recent posts and not all posts. You should therefore be using the recent_posts methods (`<% @user.recent_posts.each do |post| %>`) you created in previous milestone in the user.rb model
```
  def recent_posts
    posts.order(created_at: :desc).limit(3)
  end
```
**See screenshot of wireframe display 3 most recents posts below**
![Screen Shot 2023-09-01 at 6 38 18 AM](https://github.com/ClintonYounge/My-Blog-App/assets/62849872/60529394-f0a0-44b0-b096-bf4ad9b62868)

Reply:
Thanks. Changed.